### PR TITLE
Changing AWS resource query target

### PR DESCRIPTION
**Pull request by ahayden**
Change the query target to get tags. This seems to work when run from within a scipoolprod instance.

### CODE REVIEW

File: linux/opt/sage/bin/make_env_vars_file.sh [hunk 1]
```diff
@@ -32,6 +32,7 @@ extract_tag_value() {
 DEPARTMENT=$(extract_tag_value Department)
 PROJECT=$(extract_tag_value Project)
 PROVISIONING_PRINCIPAL_ARN=$(extract_tag_value 'aws:servicecatalog:provisioningPrincipalArn')
+ACCESS_APPROVED_ROLEID=$(extract_tag_value AccessApprovedRoleId)
```
Comment (zaro0508) [inline linux/opt/sage/bin/make_env_vars_file.sh:35]: any reason why you changed name from PRODUCT_ACCESS_APPROVED_ROLEID to ACCESS_APPROVED_ROLEID?

File: linux/opt/sage/bin/make_env_vars_file.sh [hunk 2]
```diff
@@ -73,10 +73,9 @@ export EC2_INSTANCE_ID=$EC2_INSTANCE_ID
...
 export DEPARTMENT=$DEPARTMENT
 export PROJECT=$PROJECT
+export ACCESS_APPROVED_ROLEID=$ACCESS_APPROVED_ROLEID
 export OWNER_EMAIL=$OWNER_EMAIL
 export OIDC_USER_ID=$PRINCIPAL_ID 
 export OIDC_USERNAME=$SYNAPSE_USERNAME
 export PRODUCT_NAME=$PRODUCT_NAME
-export PRODUCT_ACCESS_APPROVED_ROLEID=$PRODUCT_ACCESS_APPROVED_ROLEID
 EOM
-chmod +x "$OUTPUT_FILE"
```
Comment (zaro0508) [inline linux/opt/sage/bin/make_env_vars_file.sh:82]: why did you remove this?  we need it.

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: linux/opt/sage/bin/make_env_vars_file.sh (hunk 1 above)
Comment (ahayden) [reply linux/opt/sage/bin/make_env_vars_file.sh:35]: I changed it because it uses the ec2 describe-tags query rather than the product query.

File: linux/opt/sage/bin/make_env_vars_file.sh (hunk 2 above)
Comment (ahayden) [reply linux/opt/sage/bin/make_env_vars_file.sh:82]: I don't think we do... the next step is just `source file.sh`, which doesn't exec it.

**Review by zaro0508 — APPROVED** [review]

File: linux/opt/sage/bin/make_env_vars_file.sh (hunk 2 above)
Comment (zaro0508) [reply linux/opt/sage/bin/make_env_vars_file.sh:82]: not all of them.  some of them just do `. instance_env_vars.sh`

**Review by zaro0508 — CHANGES_REQUESTED** [review]

**Review by zaro0508 — APPROVED** [review]
